.eslintrc

```diff
@@ -1,61 +0,0 @@
-{
-  "root": true,
-  "env": { "browser": true, "node": true },
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "project": ["./tsconfig.json", "./tsconfig.tools.json"]
-  },
-  "plugins": ["@typescript-eslint", "prettier", "simple-import-sort", "import"],
-  "ignorePatterns": ["node_modules/", "tests/qunit/"],
-  "rules": {
-    "no-console": "warn",
-    "prettier/prettier": "warn",
-    "simple-import-sort/imports": "warn",
-    "simple-import-sort/exports": "warn",
-    "import/first": "error",
-    "import/no-duplicates": "error",
-    "import/newline-after-import": "warn",
-    "camelcase": "warn"
-  },
-  "overrides": [
-    {
-      "files": ["**/*.ts"],
-      "extends": [
-        "eslint:recommended",
-        "prettier",
-        "plugin:@typescript-eslint/eslint-recommended",
-        "plugin:@typescript-eslint/recommended"
-      ],
-      "rules": {
-        "@typescript-eslint/no-inferrable-types": "off",
-        "simple-import-sort/imports": [
-          "warn",
-          {
-            "groups": [
-              // Any import that starts with vex goes next.
```

Comment:
Please restore this comment (and the ones below, up until line 40 from the .eslintrc file).

Reply:
done